I 03-17 20:25:42 controller.py:59] DAG:
I 03-17 20:25:42 controller.py:59] [Task<name=compute-text-vectors>(run='# Initialize and dow...')
I 03-17 20:25:42 controller.py:59]   resources: AWS(cpus=4, {'L4': 1})]
I 03-17 20:25:42 controller.py:195] Submitted managed job 298 (task: 0, name: 'compute-text-vectors'); SKYPILOT_TASK_ID: sky-managed-2025-03-17-20-25-42-107243_compute-text-vectors_298-0
I 03-17 20:25:42 controller.py:199] Started monitoring.
I 03-17 20:25:42 state.py:451] Launching the spot cluster...
I 03-17 20:25:42 storage.py:1058] Storage type StoreType.S3 already exists.
<sky-payload>"<rich_init>[bold cyan]Optimizing[/]</rich_init>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Optimizing[/]</rich_start>"</sky-payload>
I 03-17 20:25:43 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 20:25:43 optimizer.py:749] [1mEstimated cost: [0m$0.8 / hour
I 03-17 20:25:43 optimizer.py:749] 
I 03-17 20:25:43 optimizer.py:884] Considered resources (1 node):
I 03-17 20:25:43 optimizer.py:952] ----------------------------------------------------------------------------------------
I 03-17 20:25:43 optimizer.py:952]  CLOUD   INSTANCE    vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 20:25:43 optimizer.py:952] ----------------------------------------------------------------------------------------
I 03-17 20:25:43 optimizer.py:952]  AWS     g6.xlarge   4       16        L4:1           us-east-1     0.80       [32m   âœ”[0m     
I 03-17 20:25:43 optimizer.py:952] ----------------------------------------------------------------------------------------
<sky-payload>"<rich_exit></rich_exit>"</sky-payload>
<sky-payload>"<rich_init>[bold cyan]Launching[/]</rich_init>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-25-42-755990/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-25-42-755990/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 20:25:43 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS us-east-1[0m (us-east-1a,us-east-1b,us-east-1c,us-east-1d).
[2mW 03-17 20:25:50 instance.py:243] create_instances: Attempt failed with An error occurred (VcpuLimitExceeded) when calling the RunInstances operation: You have requested more vCPU capacity than your current vCPU limit of 64 allows for the instance bucket that the specified instance type belongs to. Please visit http://aws.amazon.com/contact-us/ec2-request to request an adjustment to this limit.[0m
[2mW 03-17 20:25:51 instance.py:243] create_instances: Attempt failed with An error occurred (VcpuLimitExceeded) when calling the RunInstances operation: You have requested more vCPU capacity than your current vCPU limit of 64 allows for the instance bucket that the specified instance type belongs to. Please visit http://aws.amazon.com/contact-us/ec2-request to request an adjustment to this limit.[0m
[2mW 03-17 20:25:52 instance.py:243] create_instances: Attempt failed with An error occurred (VcpuLimitExceeded) when calling the RunInstances operation: You have requested more vCPU capacity than your current vCPU limit of 64 allows for the instance bucket that the specified instance type belongs to. Please visit http://aws.amazon.com/contact-us/ec2-request to request an adjustment to this limit.[0m
W 03-17 20:25:53 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in all zones in us-east-1 for {AWS(cpus=4, {'L4': 1})}. 
W 03-17 20:25:53 cloud_vm_ray_backend.py:2097] 
W 03-17 20:25:53 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-25-42-755990/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 20:25:53 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 20:25:53 optimizer.py:749] [1mEstimated cost: [0m$0.8 / hour
I 03-17 20:25:53 optimizer.py:749] 
I 03-17 20:25:53 optimizer.py:884] Considered resources (1 node):
I 03-17 20:25:53 optimizer.py:952] ----------------------------------------------------------------------------------------
I 03-17 20:25:53 optimizer.py:952]  CLOUD   INSTANCE    vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 20:25:53 optimizer.py:952] ----------------------------------------------------------------------------------------
I 03-17 20:25:53 optimizer.py:952]  AWS     g6.xlarge   4       16        L4:1           us-east-2     0.80       [32m   âœ”[0m     
I 03-17 20:25:53 optimizer.py:952] ----------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-25-42-755990/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-25-42-755990/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 20:25:53 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS us-east-2[0m (us-east-2a,us-east-2b,us-east-2c).
[2mW 03-17 20:25:56 instance.py:243] create_instances: Attempt failed with An error occurred (VcpuLimitExceeded) when calling the RunInstances operation: You have requested more vCPU capacity than your current vCPU limit of 128 allows for the instance bucket that the specified instance type belongs to. Please visit http://aws.amazon.com/contact-us/ec2-request to request an adjustment to this limit.[0m
[2mW 03-17 20:26:02 instance.py:243] create_instances: Attempt failed with An error occurred (VcpuLimitExceeded) when calling the RunInstances operation: You have requested more vCPU capacity than your current vCPU limit of 128 allows for the instance bucket that the specified instance type belongs to. Please visit http://aws.amazon.com/contact-us/ec2-request to request an adjustment to this limit.[0m
[2mW 03-17 20:26:03 instance.py:243] create_instances: Attempt failed with An error occurred (VcpuLimitExceeded) when calling the RunInstances operation: You have requested more vCPU capacity than your current vCPU limit of 128 allows for the instance bucket that the specified instance type belongs to. Please visit http://aws.amazon.com/contact-us/ec2-request to request an adjustment to this limit.[0m
W 03-17 20:26:04 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in all zones in us-east-2 for {AWS(cpus=4, {'L4': 1})}. 
W 03-17 20:26:04 cloud_vm_ray_backend.py:2097] 
W 03-17 20:26:04 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-25-42-755990/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 20:26:04 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 20:26:04 optimizer.py:749] [1mEstimated cost: [0m$0.8 / hour
I 03-17 20:26:04 optimizer.py:749] 
I 03-17 20:26:04 optimizer.py:884] Considered resources (1 node):
I 03-17 20:26:04 optimizer.py:952] ----------------------------------------------------------------------------------------
I 03-17 20:26:04 optimizer.py:952]  CLOUD   INSTANCE    vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 20:26:04 optimizer.py:952] ----------------------------------------------------------------------------------------
I 03-17 20:26:04 optimizer.py:952]  AWS     g6.xlarge   4       16        L4:1           us-west-2     0.80       [32m   âœ”[0m     
I 03-17 20:26:04 optimizer.py:952] ----------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-25-42-755990/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-25-42-755990/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 20:26:04 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS us-west-2[0m (us-west-2a,us-west-2b,us-west-2c,us-west-2d).
[2mW 03-17 20:26:15 instance.py:243] create_instances: Attempt failed with An error occurred (VcpuLimitExceeded) when calling the RunInstances operation: You have requested more vCPU capacity than your current vCPU limit of 64 allows for the instance bucket that the specified instance type belongs to. Please visit http://aws.amazon.com/contact-us/ec2-request to request an adjustment to this limit.[0m
[2mW 03-17 20:26:22 instance.py:243] create_instances: Attempt failed with An error occurred (VcpuLimitExceeded) when calling the RunInstances operation: You have requested more vCPU capacity than your current vCPU limit of 64 allows for the instance bucket that the specified instance type belongs to. Please visit http://aws.amazon.com/contact-us/ec2-request to request an adjustment to this limit.[0m
[2mW 03-17 20:26:36 instance.py:243] create_instances: Attempt failed with An error occurred (VcpuLimitExceeded) when calling the RunInstances operation: You have requested more vCPU capacity than your current vCPU limit of 64 allows for the instance bucket that the specified instance type belongs to. Please visit http://aws.amazon.com/contact-us/ec2-request to request an adjustment to this limit.[0m
[2mW 03-17 20:27:17 instance.py:243] create_instances: Attempt failed with An error occurred (VcpuLimitExceeded) when calling the RunInstances operation: You have requested more vCPU capacity than your current vCPU limit of 64 allows for the instance bucket that the specified instance type belongs to. Please visit http://aws.amazon.com/contact-us/ec2-request to request an adjustment to this limit.[0m
W 03-17 20:27:18 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in all zones in us-west-2 for {AWS(cpus=4, {'L4': 1})}. 
W 03-17 20:27:18 cloud_vm_ray_backend.py:2097] 
W 03-17 20:27:18 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-25-42-755990/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 20:27:18 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 20:27:18 optimizer.py:749] [1mEstimated cost: [0m$0.9 / hour
I 03-17 20:27:18 optimizer.py:749] 
I 03-17 20:27:18 optimizer.py:884] Considered resources (1 node):
I 03-17 20:27:18 optimizer.py:952] ----------------------------------------------------------------------------------------
I 03-17 20:27:18 optimizer.py:952]  CLOUD   INSTANCE    vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 20:27:18 optimizer.py:952] ----------------------------------------------------------------------------------------
I 03-17 20:27:18 optimizer.py:952]  AWS     g6.xlarge   4       16        L4:1           eu-north-1    0.85       [32m   âœ”[0m     
I 03-17 20:27:18 optimizer.py:952] ----------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-25-42-755990/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-25-42-755990/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 20:27:18 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS eu-north-1[0m (eu-north-1a,eu-north-1b).
[2mW 03-17 20:27:26 instance.py:243] create_instances: Attempt failed with An error occurred (VcpuLimitExceeded) when calling the RunInstances operation: You have requested more vCPU capacity than your current vCPU limit of 64 allows for the instance bucket that the specified instance type belongs to. Please visit http://aws.amazon.com/contact-us/ec2-request to request an adjustment to this limit.[0m
[2mW 03-17 20:27:28 instance.py:243] create_instances: Attempt failed with An error occurred (VcpuLimitExceeded) when calling the RunInstances operation: You have requested more vCPU capacity than your current vCPU limit of 64 allows for the instance bucket that the specified instance type belongs to. Please visit http://aws.amazon.com/contact-us/ec2-request to request an adjustment to this limit.[0m
W 03-17 20:27:29 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in all zones in eu-north-1 for {AWS(cpus=4, {'L4': 1})}. 
W 03-17 20:27:29 cloud_vm_ray_backend.py:2097] 
W 03-17 20:27:29 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-25-42-755990/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 20:27:29 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 20:27:29 optimizer.py:749] [1mEstimated cost: [0m$0.9 / hour
I 03-17 20:27:29 optimizer.py:749] 
I 03-17 20:27:29 optimizer.py:884] Considered resources (1 node):
I 03-17 20:27:29 optimizer.py:952] -----------------------------------------------------------------------------------------
I 03-17 20:27:29 optimizer.py:952]  CLOUD   INSTANCE    vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE    COST ($)   CHOSEN   
I 03-17 20:27:29 optimizer.py:952] -----------------------------------------------------------------------------------------
I 03-17 20:27:29 optimizer.py:952]  AWS     g6.xlarge   4       16        L4:1           ca-central-1   0.89       [32m   âœ”[0m     
I 03-17 20:27:29 optimizer.py:952] -----------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-25-42-755990/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-25-42-755990/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 20:27:29 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ca-central-1[0m (ca-central-1a,ca-central-1b).
[2mW 03-17 20:27:32 instance.py:243] create_instances: Attempt failed with An error occurred (VcpuLimitExceeded) when calling the RunInstances operation: You have requested more vCPU capacity than your current vCPU limit of 64 allows for the instance bucket that the specified instance type belongs to. Please visit http://aws.amazon.com/contact-us/ec2-request to request an adjustment to this limit.[0m
[2mW 03-17 20:27:33 instance.py:243] create_instances: Attempt failed with An error occurred (VcpuLimitExceeded) when calling the RunInstances operation: You have requested more vCPU capacity than your current vCPU limit of 64 allows for the instance bucket that the specified instance type belongs to. Please visit http://aws.amazon.com/contact-us/ec2-request to request an adjustment to this limit.[0m
W 03-17 20:27:33 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in all zones in ca-central-1 for {AWS(cpus=4, {'L4': 1})}. 
W 03-17 20:27:33 cloud_vm_ray_backend.py:2097] 
W 03-17 20:27:33 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-25-42-755990/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 20:27:33 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 20:27:33 optimizer.py:749] [1mEstimated cost: [0m$1.0 / hour
I 03-17 20:27:33 optimizer.py:749] 
I 03-17 20:27:33 optimizer.py:884] Considered resources (1 node):
I 03-17 20:27:33 optimizer.py:952] ----------------------------------------------------------------------------------------
I 03-17 20:27:33 optimizer.py:952]  CLOUD   INSTANCE    vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 20:27:33 optimizer.py:952] ----------------------------------------------------------------------------------------
I 03-17 20:27:33 optimizer.py:952]  AWS     g6.xlarge   4       16        L4:1           ap-south-1    0.97       [32m   âœ”[0m     
I 03-17 20:27:33 optimizer.py:952] ----------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-25-42-755990/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-25-42-755990/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 20:27:33 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ap-south-1[0m (ap-south-1a,ap-south-1b).
[2mW 03-17 20:27:42 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): We currently do not have sufficient g6.xlarge capacity in the Availability Zone you requested (ap-south-1a). Our system will be working on provisioning additional capacity. You can currently get g6.xlarge capacity by not specifying an Availability Zone in your request or choosing ap-south-1b.[0m
[2mW 03-17 20:27:45 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): We currently do not have sufficient g6.xlarge capacity in the Availability Zone you requested (ap-south-1b). Our system will be working on provisioning additional capacity. You can currently get g6.xlarge capacity by not specifying an Availability Zone in your request or choosing ap-south-1a.[0m
W 03-17 20:27:46 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in all zones in ap-south-1 for {AWS(cpus=4, {'L4': 1})}. 
W 03-17 20:27:46 cloud_vm_ray_backend.py:2097] 
W 03-17 20:27:46 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-25-42-755990/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 20:27:46 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 20:27:46 optimizer.py:749] [1mEstimated cost: [0m$1.0 / hour
I 03-17 20:27:46 optimizer.py:749] 
I 03-17 20:27:46 optimizer.py:884] Considered resources (1 node):
I 03-17 20:27:46 optimizer.py:952] -------------------------------------------------------------------------------------------
I 03-17 20:27:46 optimizer.py:952]  CLOUD   INSTANCE    vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE      COST ($)   CHOSEN   
I 03-17 20:27:46 optimizer.py:952] -------------------------------------------------------------------------------------------
I 03-17 20:27:46 optimizer.py:952]  AWS     g6.xlarge   4       16        L4:1           ap-northeast-2   0.99       [32m   âœ”[0m     
I 03-17 20:27:46 optimizer.py:952] -------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-25-42-755990/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-25-42-755990/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 20:27:46 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ap-northeast-2[0m (ap-northeast-2a,ap-northeast-2d).
[2mW 03-17 20:27:55 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): We currently do not have sufficient g6.xlarge capacity in the Availability Zone you requested (ap-northeast-2a). Our system will be working on provisioning additional capacity. You can currently get g6.xlarge capacity by not specifying an Availability Zone in your request or choosing ap-northeast-2d.[0m
<sky-payload>"<rich_update>[bold cyan]Launching - Checking instance status[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-25-42-755990/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching - Waiting for SSH access[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-25-42-755990/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 20:28:17 provisioner.py:451] [2mâ””â”€â”€ [0m[2mInstance is up.[0m
<sky-payload>"<rich_update>[bold cyan]Preparing SkyPilot runtime (1/3 - initializing)[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-25-42-755990/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Preparing SkyPilot runtime (2/3 - dependencies)[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-25-42-755990/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Preparing SkyPilot runtime (3/3 - runtime)[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-25-42-755990/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching - Checking instance status[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-25-42-755990/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 20:28:46 provisioner.py:632] [0m[32mâœ“ Cluster launched: compute-text-vectors-298.[0m  [2mView logs: sky api logs -l sky-2025-03-17-20-25-42-755990/provision.log[0m
<sky-payload>"<rich_exit></rich_exit>"</sky-payload>
I 03-17 20:28:46 execution.py:332] [0mâš™ï¸Ž Syncing files.
<sky-payload>"<rich_init>[bold cyan]Syncing files[/]</rich_init>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Syncing files[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Syncing files[/]</rich_start>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Syncing file mounts[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-25-42-755990/file_mounts.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 20:28:47 backend_utils.py:1315]   [2mSyncing (to 1 node): s3://skypilot-filemounts-root-726f1974-dt79ekus/job-dt79ekus/workdir -> ~/sky_workdir[0m
I 03-17 20:28:50 cloud_vm_ray_backend.py:4801] [0m[32mâœ“ Synced file_mounts.[0m  [2mView logs: sky api logs -l sky-2025-03-17-20-25-42-755990/file_mounts.log[0m
<sky-payload>"<rich_update>[bold cyan]Mounting 1 storage[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-25-42-755990/storage_mounts.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 20:28:50 backend_utils.py:1315]   [2mMounting (to 1 node): s3://sky-text-embeddings-multi-cluster-final -> /output[0m
I 03-17 20:28:54 cloud_vm_ray_backend.py:4892] [0m[32mâœ“ Storage mounted.[0m  [2mView logs: sky api logs -l sky-2025-03-17-20-25-42-755990/storage_mounts.log[0m
<sky-payload>"<rich_exit></rich_exit>"</sky-payload>
<sky-payload>"<rich_init>[bold cyan]Running setup[/]</rich_init>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Running setup[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Running setup[/]</rich_start>"</sky-payload>
<sky-payload>"<rich_stop></rich_stop>"</sky-payload>
I 03-17 20:28:57 cloud_vm_ray_backend.py:3357] [0m[32mâœ“ Setup detached.[0m
<sky-payload>"<rich_exit></rich_exit>"</sky-payload>
<sky-payload>"<rich_init>[bold cyan]Submitting job[/]</rich_init>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Submitting job[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Submitting job[/]</rich_start>"</sky-payload>
I 03-17 20:29:00 cloud_vm_ray_backend.py:3460] [0mâš™ï¸Ž Job submitted, ID: 1
<sky-payload>"<rich_stop></rich_stop>"</sky-payload>
<sky-payload>"<rich_exit></rich_exit>"</sky-payload>

[?25hI 03-17 20:29:00 recovery_strategy.py:289] Managed job cluster launched.
I 03-17 20:29:05 utils.py:131] === Checking the job status... ===
I 03-17 20:29:06 utils.py:137] Job status: JobStatus.SETTING_UP
I 03-17 20:29:06 utils.py:140] ==================================
I 03-17 20:29:07 state.py:471] Job started.
I 03-17 20:29:27 utils.py:131] === Checking the job status... ===
I 03-17 20:29:28 utils.py:137] Job status: JobStatus.SETTING_UP
I 03-17 20:29:28 utils.py:140] ==================================
I 03-17 20:29:48 utils.py:131] === Checking the job status... ===
I 03-17 20:29:49 utils.py:137] Job status: JobStatus.SETTING_UP
I 03-17 20:29:49 utils.py:140] ==================================
I 03-17 20:30:09 utils.py:131] === Checking the job status... ===
I 03-17 20:30:10 utils.py:137] Job status: JobStatus.SETTING_UP
I 03-17 20:30:10 utils.py:140] ==================================
I 03-17 20:30:30 utils.py:131] === Checking the job status... ===
I 03-17 20:30:31 utils.py:137] Job status: JobStatus.SETTING_UP
I 03-17 20:30:31 utils.py:140] ==================================
I 03-17 20:30:51 utils.py:131] === Checking the job status... ===
I 03-17 20:30:52 utils.py:137] Job status: JobStatus.SETTING_UP
I 03-17 20:30:52 utils.py:140] ==================================
I 03-17 20:31:12 utils.py:131] === Checking the job status... ===
I 03-17 20:31:13 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:31:13 utils.py:140] ==================================
I 03-17 20:31:34 utils.py:131] === Checking the job status... ===
I 03-17 20:31:35 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:31:35 utils.py:140] ==================================
I 03-17 20:31:55 utils.py:131] === Checking the job status... ===
I 03-17 20:31:56 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:31:56 utils.py:140] ==================================
I 03-17 20:32:16 utils.py:131] === Checking the job status... ===
I 03-17 20:32:17 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:32:17 utils.py:140] ==================================
I 03-17 20:32:37 utils.py:131] === Checking the job status... ===
I 03-17 20:32:38 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:32:38 utils.py:140] ==================================
I 03-17 20:32:58 utils.py:131] === Checking the job status... ===
I 03-17 20:32:59 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:32:59 utils.py:140] ==================================
I 03-17 20:33:19 utils.py:131] === Checking the job status... ===
I 03-17 20:33:20 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:33:20 utils.py:140] ==================================
I 03-17 20:33:41 utils.py:131] === Checking the job status... ===
I 03-17 20:33:42 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:33:42 utils.py:140] ==================================
I 03-17 20:34:02 utils.py:131] === Checking the job status... ===
I 03-17 20:34:03 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:34:03 utils.py:140] ==================================
I 03-17 20:34:23 utils.py:131] === Checking the job status... ===
I 03-17 20:34:24 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:34:24 utils.py:140] ==================================
I 03-17 20:34:44 utils.py:131] === Checking the job status... ===
I 03-17 20:34:45 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:34:45 utils.py:140] ==================================
I 03-17 20:35:05 utils.py:131] === Checking the job status... ===
I 03-17 20:35:10 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:35:10 utils.py:140] ==================================
I 03-17 20:35:30 utils.py:131] === Checking the job status... ===
I 03-17 20:35:34 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:35:34 utils.py:140] ==================================
I 03-17 20:35:54 utils.py:131] === Checking the job status... ===
I 03-17 20:35:55 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:35:55 utils.py:140] ==================================
I 03-17 20:36:15 utils.py:131] === Checking the job status... ===
I 03-17 20:36:16 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:36:16 utils.py:140] ==================================
I 03-17 20:36:36 utils.py:131] === Checking the job status... ===
I 03-17 20:36:37 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:36:37 utils.py:140] ==================================
I 03-17 20:36:58 utils.py:131] === Checking the job status... ===
I 03-17 20:36:59 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:36:59 utils.py:140] ==================================
I 03-17 20:37:19 utils.py:131] === Checking the job status... ===
I 03-17 20:37:20 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:37:20 utils.py:140] ==================================
I 03-17 20:37:40 utils.py:131] === Checking the job status... ===
I 03-17 20:37:41 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:37:41 utils.py:140] ==================================
I 03-17 20:38:01 utils.py:131] === Checking the job status... ===
I 03-17 20:38:02 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:38:02 utils.py:140] ==================================
I 03-17 20:38:22 utils.py:131] === Checking the job status... ===
I 03-17 20:38:23 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:38:23 utils.py:140] ==================================
I 03-17 20:38:43 utils.py:131] === Checking the job status... ===
I 03-17 20:38:44 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:38:44 utils.py:140] ==================================
I 03-17 20:39:04 utils.py:131] === Checking the job status... ===
I 03-17 20:39:05 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:39:05 utils.py:140] ==================================
I 03-17 20:39:25 utils.py:131] === Checking the job status... ===
I 03-17 20:39:26 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:39:26 utils.py:140] ==================================
I 03-17 20:39:46 utils.py:131] === Checking the job status... ===
I 03-17 20:39:47 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:39:47 utils.py:140] ==================================
I 03-17 20:40:07 utils.py:131] === Checking the job status... ===
I 03-17 20:40:08 utils.py:137] Job status: JobStatus.SUCCEEDED
I 03-17 20:40:08 utils.py:140] ==================================
I 03-17 20:40:09 state.py:561] Job succeeded.
I 03-17 20:40:09 controller.py:238] Managed job 298 (task: 0) SUCCEEDED. Cleaning up the cluster compute-text-vectors-298.
E 03-17 20:40:12 controller_utils.py:376] Failed to find the logs for the user program at ~/sky_logs/managed_jobs/sky-2025-03-17-20-25-42-755990/run.log.
I 03-17 20:40:12 controller.py:117] 
I 03-17 20:40:12 controller.py:117] == End of logs (ID: 298) ==
I 03-17 20:40:20 state.py:656] Cancellation skipped, job is already terminal
I 03-17 20:40:20 state.py:677] Cancellation skipped, job is not CANCELLING
I 03-17 20:40:21 controller.py:552] Killing controller process 157019.
I 03-17 20:40:21 controller.py:560] Controller process 157019 killed.
I 03-17 20:40:21 controller.py:562] Cleaning up any cluster for job 298.
I 03-17 20:40:22 storage.py:699] Verifying bucket for storage skypilot-filemounts-root-726f1974-dt79ekus
<sky-payload>"<rich_init>[bold cyan]Deleting S3 bucket [green]skypilot-filemounts-root-726f1974-dt79ekus[/][/]</rich_init>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Deleting S3 bucket [green]skypilot-filemounts-root-726f1974-dt79ekus[/][/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Deleting S3 bucket [green]skypilot-filemounts-root-726f1974-dt79ekus[/][/]</rich_start>"</sky-payload>
<sky-payload>"<rich_exit></rich_exit>"</sky-payload>
I 03-17 20:40:23 storage.py:1505] [32mDeleted S3 bucket skypilot-filemounts-root-726f1974-dt79ekus.[0m
I 03-17 20:40:23 controller.py:571] Cluster of managed job 298 has been cleaned up.
